#!/usr/bin/env python
PACKAGE = "offboard_ctrl"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("d0", double_t, 0, "d0 value",    1.5, 0,   3)
gen.add("k0", double_t, 0, "k0 value",    0.01, 0,   2)
gen.add("thrust_scale", double_t, 0, "thrust_scale value",    0.035, 0,   1)
gen.add("manual_trj_switch",bool_t, 0,"switch to send manual trajectory", False)
gen.add("target_x", double_t, 0, "manual trj target x",    0.00, 0,   5)
gen.add("target_y", double_t, 0, "manual trj target y",    0.00, 0,   5)
gen.add("target_z", double_t, 0, "manual trj target z",    0.00, 0,   5)

exit(gen.generate(PACKAGE, "offboard_ctrl", "dyn_params"))